### DISCUSSION TITLE
How to use "utc versions" of functions?

### DISCUSSION THREAD

**Discussion by okuRaku** [js-fns/date-fns#2647] [Q&A]
[docs/gettingStarted.md](https://github.com/date-fns/date-fns/blob/master/docs/gettingStarted.md) states:

> // UTC variation:
import addDays from 'date-fns/utc/addDays'

How does one use these "utc versions" of functions with a require() statement in a nodejs app?  I do not see them in node_modules/date-fns anyway but I assume there must be something I don't understand.

Thanks

**Comment by leveregenick** [comment]
I'm having the same issue. Maybe it's still in development and hasn't been merged?

**Reply by colemars** [reply]
I don't understand why they would put it in the docs then lmfao. Bizarre.
